docs/learn/security-policy-reporting.rst

```diff
@@ -0,0 +1,47 @@
+Security Policy Reporting
+=========================
+
+Sentry provides the ability to collect information on `Content-Security-Policy (CSP)
+<https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy>`_ violations,
+as well as `Expect-CT <https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Expect-CT>`_
+and `HTTP Public Key Pinning (HPKP) <https://developer.mozilla.org/en-US/docs/Web/HTTP/Public_Key_Pinning>`_
+failures by setting the proper HTTP header which results in violation/failure to be sent to
+Sentry endpoint specified in `report-uri`.
+
+The integration process consists of configuring the the appropriate header with
+your project key's Security Header endpoint found at
+**Project Settings > Client Keys (DSN) > Expand**.
+
+Content-Security-Policy
+-----------------------
+
+The `CSP <https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy>`_
+response header allows for control over resources can be loaded.
+
+Configure the `Content-Security-Policy` header with relevant policies and report-uri.
+
+For example, in Python this looks like the following::
+
+    Content-Security-Policy: ...; report-uri https://sentry.io/api/<project>/csp-report/?sentry_key=<key>
```

Comment:
This uri should also just be `/security/`. `/security/` is to replace all of them under the same URL, and we don't need a specific `/csp-report/` anymore.